source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -249,13 +249,13 @@ namespace AdaptiveCardsSharedModelUnitTest
         TEST_METHOD(Rule11_12Test_EscapeTest)
         {
             MarkDownParser parser("foo *\\**");
-            Assert::AreEqual<std::string>("<p>foo <em>*</em></p>", parser.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <em>\\</em>*</p>", parser.TransformToHtml());
 
             MarkDownParser parser2("foo **\\***");
-            Assert::AreEqual<std::string>("<p>foo <strong>*</strong></p>", parser2.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <strong>\\</strong>*</p>", parser2.TransformToHtml());
 
             MarkDownParser parser3("foo __\\___");
-            Assert::AreEqual<std::string>("<p>foo <strong>_</strong></p>", parser3.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <strong>\\</strong>_</p>", parser3.TransformToHtml());
```

Comment:
@jwoo-msft @paulcam206 should we use [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal) here?:)

Just for example:
```suggestion
            Assert::AreEqual<std::string>(R"<p>foo <strong>(\)</strong>_</p>", parser3.TransformToHtml());
```
We can also do `R"(raw_string)"` for more readability :)

Reply:
@beervoley 
it looks good to me.